docs/csharp/language-reference/operators/operator-overloading.md

```diff
@@ -16,7 +16,8 @@ A user-defined type can overload a predefined C# operator. That is, a type can p
 Use the `operator` keyword to declare an operator. An operator declaration must satisfy the following rules:
 
 - It includes both a `public` and a `static` modifier.
-- A unary operator has one input parameter. A binary operator has two input parameters. In each case, at least one parameter must have type `T` or `T?` where `T` is the type that contains the operator declaration.
+- A unary operator has one input parameter; a binary operator has two input parameters. In each case at least one parameter must be of type `T` or `T?`, where `T` is the type that contains the operator declaration.
+- It returns either a `boolean` value (comparison operators) or an instance of the type defining the operator (all other operators).
```

Comment:
That is not correct statement. As the [following example (sharplab)](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQxAgrgOwD4AEBMAjALABQZuAzAAQJwQAmA9pgDYCe1sGAxjNQDEEEPgEsWZAN5lqs6gAcEogG4R4teszadRmfpnRgA3DLmKVauBsYsO1XfwZxMJ8qTnUqg4WJYAKB2oDMEQ1JmR7PWonTCYwXTCEAEpTWWl3DzlRADNqPxi4hJhw6gBeUuoABhSMzOp0urqYAAsEJgB3ILhOgEEEAHNDZxgAUQAPHjh5GHFMPwAiABFnQsxE6h4ITFj+YCsAL0QmADp5pCCIEKZs/JX4teLkpNdG6gBfVMzgsqCh4UeXo0Yj8CvdEoDZB83B4vLhCAA2SL8JjyUKPagAaj8QhEMxY1AgSTKAD5qIQIZ4aHDEXDKtQUWiSgBabE+PGYAlE0qk+bcLb9ODzCmwhHRJjoYCsKwM/4lLE43wciDnBXs6jALmkgCyamaxwACgBJVyfLxMZSIJROTyEOkAFSYAGUYEpMP0/JrqAASeaSYJvagAenqMTeQrIUIoVNFBGoAHlUbKEHGLQhWExGLp+lJTfh8J8Go1VAgCT9MN1vLjZn4AKznAAsz0+HmL6rLFdV1cI53wTdqmThAE4/EzCUY5IHg+KYPJ0KguC7+XBm3Ih34IJj1c9ZJP6XPZ/PKMdCPXCDXB/h4TXKDWAByDgDsg8onyhbyAA==) shows, return type can be any:

```csharp
using System;

public readonly struct Fraction
{
    private readonly int num;
    private readonly int den;

    public Fraction(int numerator, int denominator)
    {
        if (denominator == 0)
        {
            throw new ArgumentException("Denominator cannot be zero.", nameof(denominator));
        }
        num = numerator;
        den = denominator;
    }

    public static int operator +(Fraction a) => 1;
    public static string operator -(Fraction a) => "strange";
    public static double operator +(Fraction a, Fraction b) => Math.PI;

    public override string ToString() => $"{num} / {den}";
}

public static class OperatorOverloading
{
    public static void Main()
    {
        var a = new Fraction(5, 4);
        var b = new Fraction(1, 2);
        Console.WriteLine(-a);   // output: strange
        Console.WriteLine(a + b);  // output: 3.141592653589793
    }
}
```

Reply:
The second item in the list is technically wrong. @pkulikov has one example. Many other operators have other rules:  member access (`x.y`), index (`x[m]`), `typeof`, and so on. Many of these aren't overloadable, but the statement is still misleading.

@IanKemp What do you want readers to learn from this statement? That will help me come up with alternative wording for this bullet point.